nginx/nginx.conf

```
@@ -17,7 +17,7 @@ http {
 
 	access_log  /etc/nginx/log/access.log  main;
 	sendfile        on;
-	keepalive_timeout  65;
-
-	 include /etc/nginx/conf.d/*.conf;	
+	keepalive_timeout  360;
+    proxy_read_timeout 600;
+    include /etc/nginx/conf.d/*.conf;	
```

Is there any rationale for those numbers or where they decided just based on trying and getting `gettxoutsetinfo` to run?

Not relevant but consistent indentation :man_shrugging: :
```suggestion
	keepalive_timeout  360;
	proxy_read_timeout 600;
	include /etc/nginx/conf.d/*.conf;	
```

---

Also might fix other gateway timeouts